tox.ini

```diff
@@ -43,12 +44,12 @@ description = Run static analysis checks
 setenv =
     PYTHONPATH = ""
 commands =
-    mypy {[vars]all_path} {posargs}
+    mypy {[vars]all_path} --exclude tests/unit/lib/charms/ {posargs}
```

Comment:
We're not:
`all_path = {[vars]src_path} {[vars]unit_test_path} {[vars]integration_test_path} {[vars]lib_path}`
I'm only excluding dummy test charms, because otherwise mypy throws `Duplicate module named "charm" (also at "src/charm.py")`.